File: Practica_1/src/gestorAplicacion/personas/Administrador.java
package gestorAplicacion.personas;

public class Administrador extends Empleado{
	
	private static final long serialVersionUID = 1L;
	private String usuario;
	private String contrasena;
	
	public Administrador(int cedula, String nombre, float sueldo, String usuario, String contrasena) {
		super(cedula, nombre, sueldo);
		this.cedula = cedula;
		this.nombre = nombre;
		this.sueldo = sueldo;
		this.usuario = usuario;
		this.contrasena = contrasena;
	}
	
	public String toString() {
		return "El administrador " + this.nombre + " identificado con el número de cedula " + this.cedula + " tiene como funciones: supervisar el local, dirigir a los baristas y hacer uso correcto del sistema al que se le dio acceso";

	}

	public String getUsuario() {
		return usuario;
	}

	public String getContrasena() {
		return contrasena;
	}

	public void setUsuario(String usuario) {
		this.usuario = usuario;
	}

	public void setContrasena(String contrasena) {
		this.contrasena = contrasena;
	}
	
	
	
	

}

File: Practica_2/personas/administrador.py
from empleado import Empleado

class Administrador(Empleado):
    def __init__(self, cedula, nombre, sueldo, usuario, contrasena):
        self._cedula = cedula
        self._nombre = nombre
        self._sueldo = sueldo
        self._usuario = usuario
        self._contrasena = contrasena
        
    def __str__(self):
        return  "El administrador " + self._nombre + " identificado con el número de cedula " + self._cedula + " tiene como funciones: supervisar el local, dirigir a los baristas y hacer uso correcto del sistema al que se le dio acceso"
    
    def getUsuario(self):
        return self._usuario
    
    def getContrasena(self):
        return self._contrasena
    
    def setUsuario(self, usuario):
        self._usuario = usuario
    
    def setContrasena(self, contrasena):
        self._contrasena = contrasena